.github/CONTRIBUTING.md

```diff
@@ -64,7 +65,7 @@ information on our formatting guidelines.
 
 ### Compatibility ###
 
-Cockatrice is currently compiled on all platforms using <kbd>C++11</kbd>.
+Cockatrice is currently compiled on all platforms using <kbd>C++20</kbd>.
 You'll notice <kbd>C++03</kbd> code throughout the codebase. Please feel free
```

Comment:
Updated the `C++20` note following https://github.com/Cockatrice/Cockatrice/pull/5244.

For the reference to old code I was not sure... better to note the former `C++11` there, or is there still `C++03` code around?